package.json

```diff
@@ -93,6 +93,7 @@
   },
   "dependencies": {
     "@ciscospark/eslint-config-base": "^0.5.0",
+    "@ciscospark/test-helper-test-users": "^1.48.0",
```

Comment:
You may have to rebase to 1.49.

Reply:
Accidentally installed this
This is actually a package in our repo 😝